src/ashell/shell-state.c

```diff
@@ -622,6 +651,36 @@ int32_t ashell_help(char *buf)
     return RET_OK;
 }
 
+void ashell_run_boot_cfg()
+{
+    fs_file_t *file;
+    size_t count;
+
+    file = fs_open_alloc("boot.cfg", "r");
+
+    // Failed to open boot.cfg
+    if (!file) {
+        return;
+    }
+
+    ssize_t size = fs_size(file);
+    if (size == 0) {
+        printk("Empty boot.cfg file");
+        fs_close_alloc(file);
+        return;
+    }
+    else if (size > 12) {
+        fs_close_alloc(file);
+        return;
+    }
+
+    char data[size+1];
+    count = fs_read(file, data, size);
+    data[size] = '\0';
+    ashell_run_javascript(data);
+    fs_close(file);
```

Comment:
I just also realized that the uart won't be up by that time.  So literally no reason to print an error because it won't show up anyway.  Removing the prints.